app/settings.py

```diff
@@ -137,3 +137,5 @@ def utcoffset_or_fail(date_value, key):
 ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS = int(
     os.getenv("ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS", "300")
 )
+
+SURVEY_TYPE = os.getenv("SURVEY_TYPE", "census")
```

Comment:
I'm struggling to get this working in my local runner after adding env variable when I use test schemas, they have hardcoded default themes. Do you know in which schemas should I test this?

Reply:
I'd try pulling in one of the census schemas manually @petechd - apologies for it being a pain